nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/WriteAheadFlowFileRepository.java

```diff
@@ -80,6 +80,15 @@
  */
 public class WriteAheadFlowFileRepository implements FlowFileRepository, SyncListener {
     private static final String FLOWFILE_REPOSITORY_DIRECTORY_PREFIX = "nifi.flowfile.repository.directory";
+    private static final String WRITE_AHEAD_LOG_IMPL = "nifi.flowfile.repository.wal.implementation";
+
+    // TODO: Update Admin Guide
```

Comment:
I think this TODO is DONE :)

Reply:
Yup, this is done :) If you get a chance to merge, please delete the comment.